apps/main-landing/src/components/superpowers-section/superpowers-section.tsx

```diff
@@ -20,16 +32,79 @@ export const SuperpowersSection = () => {
           </p>
         </div>
         <div className="flex flex-col items-start justify-center gap-6 lg:grid lg:grid-cols-[repeat(3,_minmax(auto,_431px))] lg:flex-row">
-          {productTilesContent.map((point) => {
-            return (
-              <ProductTile
-                key={point.productName}
-                productName={point.productName}
-                tileTitle={point.tileTitle}
-                tileListPoints={point.tilePoints}
-              />
-            );
-          })}
+          <ProductTile
+            {...EXTENSION_INFO}
+            name={EXTENSION_INFO.name}
+            title={EXTENSION_INFO.title}
+            features={EXTENSION_INFO.features}
```

Comment:
probably you don't need to explicitly provide name/title/features since you spreaded this obj